3/3-Core-Structures.md

```diff
@@ -58,6 +58,30 @@ separately in [Part 1.2, Extension Structures](4-Extension-Structures.md).
     - [EmailAddress](#opencannabis.contact.EmailAddress)
     - [Website](#opencannabis.contact.Website)
 - `products`: High-level concrete product structures.
+    - [Merchandise](#opencannabis.products.Merchandise)
+    - [MerchandiseFlag](#opencannabis.products.MerchandiseFlag)
+    - [MerchandiseType](#opencannabis.products.MerchandiseType)
+    - [Apothecary](#opencannabis.products.Apothecary)
+    - [ApothecaryType](#opencannabis.products.ApothecaryType)
+    - [Flower](#opencannabis.products.Flower)
+    - [Edible](#opencannabis.products.Edible)
+    - [EdibleIngredient](#opencannabis.products.EdibleIngredient)
```

Comment:
What exactly is edible ingredient? This is the first I am seeing of this in the model.

Reply:
References ingredients in the edible